cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -358,6 +366,9 @@ public TwillPreparer withConfiguration(Map<String, String> config) {
     if (config.containsKey(KubeTwillRunnerService.RUNTIME_CLEANUP_DISABLED)) {
       runtimeCleanupDisabled = Boolean.parseBoolean(config.get(KubeTwillRunnerService.RUNTIME_CLEANUP_DISABLED));
     }
+    if (config.containsKey(MasterOptionConstants.RUNTIME_NAMESPACE)) {
+      cdapRuntimeNamespace = config.get(MasterOptionConstants.RUNTIME_NAMESPACE);
```

Comment:
how is this different than the programRuntimeNamespace above?

Reply:
programRuntimeNamespace is the kubernetes namespace in which the program runs in which does not necessarily match the CDAP namespace